### DISCUSSION TITLE
circular search param is available

### DISCUSSION THREAD

**Discussion by MALKARAJ** [osm-search/Nominatim#2842] [Q&A]
Hi team
is there any param for circular search  like (boundary.circle.lat , boundary.circle.lon , boundary,circle.radius) in nomination I couldn't find those in nomi official documentation . kindly provide some help on this

**Comment by MALKARAJ** [comment]
Hi any body help me in this

**Comment by mtmail** [comment]
Nominatim doesn’t have these parameters. You can use ‘viewbox’ to specify a rectangular box plus ‘bounds=1’. It requires a bit of extra work from you to calculate those from a center point and radius.
